src/porepy/examples/tracer_flow.py

```diff
@@ -0,0 +1,274 @@
+"""Module containing a simple tracer flow setup, modelled as a single phase, 2-component
+flow."""
+
+from __future__ import annotations
+
+from typing import Literal, Sequence, TypedDict, cast
+
+import numpy as np
+
+import porepy as pp
+from porepy.applications.material_values.fluid_values import water as _water
+from porepy.applications.md_grids.domains import nd_cube_domain
+from porepy.compositional.compositional_mixins import CompositionalVariables
+from porepy.models.compositional_flow import (
+    BoundaryConditionsMulticomponent,
+    ComponentMassBalanceEquations,
+    InitialConditionsFractions,
+)
+from porepy.models.fluid_mass_balance import SinglePhaseFlow
+
+
+class WaterDict(TypedDict):
+    """For mypy when using **water in combination with FluidComponent."""
+
+    compressibility: float
+    density: float
+    specific_heat_capacity: float
+    thermal_conductivity: float
+    thermal_expansion: float
+    viscosity: float
+    name: str
+
+
+# Casting this to help mypy
+water = cast(WaterDict, _water)
+
+
+class DomainX(pp.PorePyModel):
+    """Unit square domain with two line fractures forming a X."""
+
+    def set_domain(self) -> None:
+        """Setting a 2d unit square as matrix."""
+        size = self.units.convert_units(1, "m")
+        self._domain = nd_cube_domain(2, size)
+
+    def set_fractures(self) -> None:
+        """Setting 2 fractures in x shape."""
+        frac_1_points = self.units.convert_units(
+            np.array([[0.2, 0.8], [0.2, 0.8]]), "m"
+        )
+        frac_1 = pp.LineFracture(frac_1_points)
+        frac_2_points = self.units.convert_units(
+            np.array([[0.2, 0.8], [0.8, 0.2]]), "m"
+        )
+        frac_2 = pp.LineFracture(frac_2_points)
+        self._fractures = [frac_1, frac_2]
+
+    def grid_type(self) -> Literal["simplex", "cartesian", "tensor_grid"]:
+        return self.params.get("grid_type", "simplex")
+
+    def meshing_arguments(self) -> dict:
+        cell_size = self.units.convert_units(0.05, "m")
+        cell_size_fracture = self.units.convert_units(0.05, "m")
+        mesh_args: dict[str, float] = {
+            "cell_size": cell_size,
+            "cell_size_fracture": cell_size_fracture,
```

Comment:
You don't need both, since they're identical.

Reply:
Removing redundant `cell_size_fracture`.